settings.d/dynflow.yml.example

```diff
@@ -1,3 +1,5 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
+:console_authorize_with_ssl_client: true
+:console_authorize_with_trusted_hosts: false
```

Comment:
IMHO it feels wrong for us to touch these at all.  It a user has configured trusted_hosts or ssl client certificates in settings.yml, they should assume it protects all of the sensitive components.  The dynflow console being one of them.

Why are we creating a second set of options @iNecas? I guessed it was to make it easier to get into the console if a troubleshooting case, and in that case, you would be concerned with turning both off.